commands/genpoke.js

```diff
@@ -6,25 +6,31 @@ const logger = require('../logs/logger.js');
 const CMD_TEMPLATE = '+genpoke [SPECIES] [LEVEL (1-20)] [NICKNAME - no spaces or special characters] [HIDDEN' +
 	' ABILITY % (as a number, 0-100)]';
 //help message
+const HELP_MESSAGE = '\n`+genpoke [species] [level (1-20)] [nickname] [Form Name] [hidden ability % (as a number, 0-100)]\n' +
+	'**FIELDS TO ADD LATER:**\n [trainerName] [private (optional: leave blank if public, put 1 if private]`' +
+	'\n\nCreates a new Pokemon when given the values above, printing it upon completion.\n\n';
```

Comment:
Add example filled out command + clarifications:

Generates a Pokemon and adds it to the database. Minimum requirements are the pokemon's species, level (1-20),and nickname.

If desired, you can specify a form and hidden ability, but unless the form is stored in our database, it will default to the base species listed at the beginning.

Example standard Pokemon:
`+genpoke Meganium 10 Meggie`
OR `+genpoke Meganium 10 Meggie Meganium 50` (generates a meganium with a 50% chance of having it's hidden ability)

Example Kantonian Vulpix:
`+genpoke Vulpix 10 VulKanto`

Example Alolan Vulpix:
`+genpoke Vulpix 10 VulAlola vulpix-alola`

After creating your Pokemon, we suggest using `+showpoke (nickname)` to preview them properly.

Reply:
Done